code/_globalvars/lists/mobs.dm

```diff
@@ -66,6 +66,12 @@ GLOBAL_LIST_EMPTY(mob_config_movespeed_type_lookup)
 
 GLOBAL_LIST_EMPTY(emote_list)
 
+GLOBAL_LIST_INIT(construct_types, list(
+	"Juggernaut" = image(icon = 'icons/mob/cult.dmi', icon_state = "juggernaut"),
+	"Wraith" = image(icon = 'icons/mob/cult.dmi', icon_state = "wraith"),
+	"Artificer" = image(icon = 'icons/mob/cult.dmi', icon_state = "artificer")
+))
```

Comment:
Name this var better, it's construct radial images, not just a list of construct types